### PR TITLE
APPDRUSTI-24 ANNOUNCEMENT page

### PR DESCRIPTION
**Pull request by BOSSHK-SPEC**

### CODE REVIEW

File: lib/Announcement.dart [hunk 1]
```diff
@@ -1,27 +1,421 @@
...
+                        new EdgeInsets.symmetric(vertical: 60.0, horizontal: 0.0),
+                        border: OutlineInputBorder(
+                            borderRadius: BorderRadius.circular(32.0)),
+                        focusedBorder: OutlineInputBorder(
+                          borderSide: BorderSide(color: Colors.black, width: 1.0),
+                        ),
+                        enabledBorder: OutlineInputBorder(
+                          borderSide: BorderSide(color: Colors.black, width: 1.0),
+                        ),
+                        hintText: 'Discription',
```
Comment (sahana-drusti) [inline lib/Announcement.dart]: spelling of Description

File: lib/Announcement.dart [hunk 2]
```diff
@@ -1,27 +1,421 @@
...
+
+                            },
+                          ), // icon is 48px widget.
+                        ),
+                        hintText: 'Attachment',
+                        labelText: 'Attachment',
+                      ),
+                      validator: (val) {
+                        if (val == null || val.isEmpty) {
+                          return 'Please enter Description';
```
Comment (sahana-drusti) [inline lib/Announcement.dart]: no need to validate attachment, it can be empty

File: lib/Announcement.dart (hunk 1 above)
Comment (BOSSHK-SPEC) [reply lib/Announcement.dart]: ok

File: lib/Announcement.dart (hunk 2 above)
Comment (BOSSHK-SPEC) [reply lib/Announcement.dart]: ok

**Pull request merged**